<launch>

    <!-- Run the map server -->
  <arg name="map_file" default="$(find tb_waiter)/maps/lab2.yaml"/>
  <!--  -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />

  
  <!-- BEGIN ROBOT 0 --> 
  <group ns="robot_0">
    <param name="tf_prefix" value="robot_0" />
    <!--- Run AMCL -->
    <include file="$(find tb_waiter)/launch/simulation/stage/5_lab_sim3/amcl/amcl_diff.launch.xml" >
    	<arg name="robot_name" value="/robot_0"/>
    	<arg name="scan_topic" value="/robot_0/base_scan"/>
    	<arg name="initial_pose_x" value="1.489"/>
    	<arg name="initial_pose_y" value="3.543"/>
    	<arg name="initial_pose_a" value="3.14"/>
    </include>
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />
    
    <include file="$(find tb_waiter)/launch/simulation/stage/include/move_base_stage.launch.xml" />
  </group>
  
  <!-- BEGIN ROBOT 1 --> 
  <group ns="robot_1">
    <param name="tf_prefix" value="robot_1" />
    <!--- Run AMCL -->
    <include file="$(find tb_waiter)/launch/simulation/stage/5_lab_sim3/amcl/amcl_diff.launch.xml" >
    	<arg name="robot_name" value="/robot_1"/>
    	<arg name="scan_topic" value="/robot_1/base_scan"/>
    	<arg name="initial_pose_x" value="1.418"/>
    	<arg name="initial_pose_y" value="1.868"/>
    	<arg name="initial_pose_a" value="3.14"/>
    </include>
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />
    
    <include file="$(find tb_waiter)/launch/simulation/stage/include/move_base_stage.launch.xml" />
  </group>
</launch>

